app/views/shared/_search.html.erb

```diff
@@ -0,0 +1,5 @@
+<form action="/catalog">
+  <img src="/icon-search.png"/>
+  <input placeholder="Search over 10,000 WGBH clips online" 
+         onchange="this.form.submit();" name="q"/>
+</form>
```

Comment:
@mccalluc The `onchange` event will fire if you type a search and then clicks or tabs away from the input. I don't think we want that. Try this maybe:
```html
<form action="/catalog">
  <input type="image" src="/icon-search.png"/ alt="Search">
  <input placeholder="Search over 10,000 WGBH clips online" name="q"/>
</form>
```